src/lib/fragments/Accordion.react.js

```diff
@@ -45,7 +54,7 @@ class Accordion extends Component {
           <div className={sectionIsOpen ? "govuk-accordion__section govuk-accordion__section--expanded" : "govuk-accordion__section"}>
             <div className="govuk-accordion__section-header">
               <h2 className="govuk-accordion__section-heading">
-                <button type="button" aria-controls={`accordion-default-content-${index}`} className="govuk-accordion__section-button" aria-expanded={sectionIsOpen} onClick={() => this.openOrCloseAccordionSection(index)}>
+                <button className="accordion-button govuk-accordion__section-button"type="button" aria-controls={`accordion-default-content-${index}`} aria-expanded={sectionIsOpen} onClick={() => this.openOrCloseAccordionSection(index)}>
```

Comment:
I'm pretty sure there needs to be a space between the " and the `type="button"` here

Reply:
I just made this change!